core/client-internal/src/test/java/tachyon/client/util/ClientTestUtils.java

```diff
@@ -28,9 +34,43 @@
    * Sets small buffer sizes so that Tachyon does not run out of heap space.
    */
   public static void setSmallBufferSizes() {
-    TachyonConf mConf = new TachyonConf();
-    mConf.set(Constants.USER_BLOCK_REMOTE_READ_BUFFER_SIZE_BYTES, "4KB");
-    mConf.set(Constants.USER_FILE_BUFFER_BYTES, "4KB");
-    ClientContext.reset(mConf);
+    ClientContext.getConf().set(Constants.USER_BLOCK_REMOTE_READ_BUFFER_SIZE_BYTES, "4KB");
+    ClientContext.getConf().set(Constants.USER_FILE_BUFFER_BYTES, "4KB");
+  }
+
+  /**
+   * Resets the {@link ClientContext} singleton.
+   *
+   * This method should only be used as a cleanup mechanism between tests. It should not be used
```

Comment:
It might be worth mentioning that reset the `ClientContext` will reinitialize it.